install_config/persistent_storage/persistent_storage_vsphere.adoc

```diff
@@ -105,16 +105,17 @@ spec:
 <1> The name of the volume. This must be how it is identified by xref:../../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[PV claims] or from pods.
 <2> The amount of storage allocated to this volume.
 <3> This defines the volume type being used (*vsphereVolume* plug-in, in this example). The `vsphereVolume` label is used to mount a vSphere VMDK volume into pods. The contents of a volume are preserved when it is unmounted. The volume type supports VMFS and VSAN datastore.
-<4> This VMDK volume must exist.
+<4> This VMDK volume must exist. The square brackets (`[]`) must be included when defining the VMDK volume.
```

Comment:
I'd say "This VMDK volume must exist, and you must include brackets (`[]`) in the volume definition."
(Style says to not call them square brackets.)